src/Backend/Modules/FormBuilder/Engine/Autocomplete.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Backend\Modules\FormBuilder\Engine;
 
+use Backend\Core\Language\Language as BL;
```

Comment:
Don't alias the `Language` class... I know it's done all over Fork but it's really pretty useless. Just use `Language::lbl` :)

Reply:
The alias is because of legacy.